validation/fpc/fpc_validation.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
```

Comment:
Moving stuff out of plugins makes sense if we do not use plugins anymore. But a `git mv` would have kept the changes smaller and kept provenance?

Reply:
btw: you also will have to update core-fpc.yaml in utils/docker-compose ...

Reply:
updated